Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Rating.hs

```diff
@@ -0,0 +1,43 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module API.UI.Rating
+  ( API,
+    handler,
+  )
+where
+
+import Domain.Action.UI.Rating hiding (getDriverRating)
+import qualified Domain.Action.UI.Rating as Domain
+import qualified Domain.Types.Person as SP
+import Environment
+import EulerHS.Prelude
+import Kernel.Types.APISuccess
+import Kernel.Types.Id (Id (..))
+import Kernel.Utils.Common
+import Servant
+import Tools.Auth
+
+type API =
+  "rating"
+    :> TokenAuth
+    :> ReqBody '[JSON] RatingReq
+    :> Post '[JSON] APISuccess
+
+handler :: FlowServer API
+handler =
+  getDriverRating
```

Comment:
this name can be something which specifies that it is customer's rating which driver is giving. It can be misleading as rating of the driver. Can be something like getCustomerRatingByDriver